examples/cpp-headless.cpp

```diff
@@ -19,30 +19,30 @@
 #define STB_IMAGE_WRITE_IMPLEMENTATION
 #include "third_party/stb_image_write.h"
 
-void normalize_depth_to_rgb(uint8_t rgb_image[640*480*3], const uint16_t depth_image[], int width, int height)
+void normalize_depth_to_rgb(uint8_t rgb_image[640 * 480 * 3], const uint16_t depth_image[], int width, int height)
 {
-    for (int i = 0; i < width * height; ++i)
+    for(int i = 0; i < width * height; ++i)
     {
-        if (auto d = depth_image[i])
+        if(auto d = depth_image[i])
         {
-            uint8_t v = d * 255 / std::numeric_limits<uint16_t>::max();
-            rgb_image[i*3 + 0] = 255 - v;
-            rgb_image[i*3 + 1] = 255 - v;
-            rgb_image[i*3 + 2] = 255 - v;
+            uint8_t v            = d * 255 / std::numeric_limits<uint16_t>::max();
```

Comment:
as I explained above, clang-format tries to align following line declaration. I'll fix it.